---
abstract: |
  Considering the fact that only small portion of bacterial species are culturable, the metagenome sequences derived from environmental samples harbour yet to be discovered insights. With the emergence of next-generation sequencing technologies the metagenomic studies has been accelerated. Not only 16S rRNA but also whole metagenome extracted from an environmental sample is subjected to analysis. Such a depth in analysis allows multiple sequence comparisons for a selected gene across metagenomes.

   In this study, we used publicly available whole metagenome sequencing data from which sequences for industrial enzymes were extracted and subjected to multiple sequence alignment. The sequence differences for a given enzyme were overlaid on top of three dimensional structure of that enzyme. 

  Our method reveals the amino acid residue changes at certain locations in select enzymes. The sequence changes we have identified are novel and has not been described in bacterial enzyme databases.

  Such an approach has potential to pave the way for better understanding of enzyme structure and protein engineering implications.

authors:
  - Seyma Ozcirak
  - Alper Yilmaz
date: "2016-07-01T00:00:00+03:00"
doi: doi.org/10.1111/febs.13807
featured: false
image:
  caption: 
  focal_point: ""
links:
- name: FEBS Journal
  url: https://febs.onlinelibrary.wiley.com/toc/17424658/2016/283/S1
publication: In *41st FEBS Congress*
publication_short: ""
publication_types:
  - "1"
slides: 
summary: 
tags:
  - Metagenome
title: Protein engineering insights from metagenomes of diverse bacteria
url_code: 
url_dataset: 
url_pdf: https://febs.onlinelibrary.wiley.com/doi/epdf/10.1111/febs.13807
url_poster: 
url_project: 
url_slides: 
url_source: 
url_video: 
---

{{% callout note %}}
Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
{{% /callout %}}
